_reports/kriti21.md

```diff
@@ -0,0 +1,125 @@
+---
+categories:
+  - GSoC
+  - 2018
+  - StatusReport
+report_year: 2018
+date: 2018-08-13
+student: Kriti Rohilla
+organisation: coala
+organisation_link: https://coala.io
+project: Git Commit Content Inspection
+project_link: https://summerofcode.withgoogle.com/projects/#5636421247827968
+tarball: https://raw.githubusercontent.com/kriti21/GSoC-2018-Report/master/ProjectPatches.tar.gz
+sha256sum: 2e42b6a0f2665ab5d7b94f0f101b18a74f4fd4a8d6ea7f59a9eaef4621107213
+mentors: >
+  [Saurav Singh](https://github.com/damngamerz),
+  [Rafid Aslam](https://github.com/refeed),
+  [Guyzmo](https://github.com/guyzmo)
+phase:
+  - Bonding: https://gitlab.com/coala/GSoC/GSoC-2018/milestones/9
+  - Phase 1: https://gitlab.com/coala/GSoC/GSoC-2018/milestones/36
+  - Phase 2: https://gitlab.com/coala/GSoC/GSoC-2018/milestones/41
+  - Phase 3: https://gitlab.com/coala/GSoC/GSoC-2018/milestones/42
+bio: >
+  I am in my pre-final year at University Institute of Engineering and Technology,
+  Panjab University. As part of my GSoC 2018 project, I worked on VCS related bears
+  in [coala-bears](https://github.com/coala/coala-bears/). The main task was to
+  detect special git commits like `git revert`, `git merge` and commits that skip
+  CI build and inspect such commits to see whether they meet the requirements of the
+  project.
+social:
+  - GitHub:
+      - username: kriti21
+      - link: https://github.com/kriti21
+  - GitLab:
+      - username: kriti21
+      - link: https://gitlab.com/kriti21
+  - Gitter:
+      - username: kriti21
+      - link: https://gitter.im/kriti21
+  - Openhub:
+      - username: kriti21
+      - link: https://www.openhub.net/accounts/kriti21
+email: kritirohilla567@gmail.com
+blog: https://kriti21.github.io/
+activity:
+  - 1:
+      - repo: projects
+      - link: https://github.com/coala/projects/pull/562/commits/52c11a2c85087e705b2a78197aed754d47221174
+      - details: >
+         commit-content-checks.md: Add issue link to project description.
+  - 2:
+      - repo: cEPs
+      - link: https://github.com/coala/cEPs/pull/114/commits/ea25f09ae08b4bc7423453460b14cb16eb3a0db6
```

Comment:
this isnt in your tarball.  it should be

and the igitt project .md is missing from this list. it should be included here, as it is hard work, and was related to your current project.  It is also "community work" which is good to include